tests/Cases/DI/RemoteAccessExtensionIntegrationTest.php

```diff
@@ -13,7 +13,7 @@
 class RemoteAccessExtensionIntegrationTest extends Tester\TestCase
 {
 	/**
-	 * @return void
+	 * @throws \Exception
```

Comment:
Missing annotation `@return`

Reply:
The return type in phpDocs is no longer needed when it's present in PHP, I'm I right?

Reply:
You are right, but same problem is with annotation `@param`. If you declare type of parameter you don't need it in the phpDoc but we still write it.